src/components/Button/Button.tsx

```diff
@@ -0,0 +1,28 @@
+export interface ReusableBtnProps {
```

Comment:
just `ButtonProps`. When I say reusable, I mean you put (most of the time) such a component inside `components` folder